tests/integration/distribution/keeper/grpc_query_test.go

```diff
@@ -124,7 +124,7 @@ func TestGRPCValidatorOutstandingRewards(t *testing.T) {
 			name:      "invalid address",
 			msg:       &types.QueryValidatorOutstandingRewardsRequest{ValidatorAddress: sdk.ValAddress("addr1_______________").String()},
 			expPass:   false,
-			expErrMsg: "validator does not exist",
+			expErrMsg: "not found",
```

Comment:
any way we can add more context to this error? not found is can mean a few things

Reply:
since the error is from collections `ErrNotFound`. It'd be something like `"collections: not found: key 'cosmos..' of type github.com/cosmos/gogoproto/cosmos.staking.v1beta1.Validator"` and we cannot provide more context here

Reply:
reverted in the latest commits